server/controllers/requests.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const UserRequests = require( "../models/UserRequests");
+
+//GET /requests: list of requests for logged in user
+exports.getRequests = async (req, res) => {
+  try {
+    const userRequests = await UserRequests.find();
```

Comment:
As is, this won't filter requests by user and get all the requests in the database. You could do something along the lines of `...find({userId: req.userId})`.